packages/design-system/src/components/toast.tsx

```diff
@@ -71,67 +66,177 @@ const StyledToast = styled(ToastPrimitive.Root, {
       animation: `${swipeOut} 150ms ease-out`,
     },
   },
+});
+
+const borderAccentBackgroundColor = "--ws-toast-border-accent-background-color";
+const backgroundColor = "--ws-toast-background-color";
+const borderColor = "--ws-toast-border-color";
+const iconColor = "--ws-toast-icon-color";
+
+const ToastVariants = styled("div", {
+  [borderAccentBackgroundColor]: theme.colors.foregroundMain,
+  [backgroundColor]: theme.colors.backgroundNeutralNotification,
+  [borderColor]: theme.colors.borderNeutral,
+  [iconColor]: theme.colors.foregroundMain,
+
   variants: {
     variant: {
-      error: {
-        background: theme.colors.red9,
-        color: theme.colors.loContrast,
+      neutral: {},
+      warning: {
+        [backgroundColor]: theme.colors.backgroundAlertNotification,
+        [borderAccentBackgroundColor]: theme.colors.backgroundAlertMain,
+        [borderColor]: theme.colors.backgroundAlertMain,
+        [iconColor]: theme.colors.backgroundAlertMain,
       },
-      success: {
-        background: theme.colors.green10,
-        color: theme.colors.loContrast,
-      },
-      blank: {
-        background: theme.colors.blue11,
-        color: theme.colors.loContrast,
+      error: {
+        [backgroundColor]: theme.colors.backgroundDestructiveNotification,
+        [borderAccentBackgroundColor]: theme.colors.backgroundDestructiveMain,
+        [borderColor]: theme.colors.backgroundDestructiveMain,
+        [iconColor]: theme.colors.foregroundDestructive,
       },
-      custom: {},
-      loading: {},
     },
   },
 });
 
+type ToastVariant = React.ComponentProps<typeof ToastVariants>["variant"];
+
+const cssVar = (name: string) => `var(${name})`;
```

Comment:
I know this is convenient, but I would prefer to either inline it or to have a single utility that is reusable, simply foc sonsistency

Reply:
To early for the utility. Just played with the idea of using variables for all variants. I don't think that everything we are playing with needs an utility